core/src/models/RecordProperty.ts

```diff
@@ -25,7 +30,10 @@ const STATE_TRANSITIONS = [
 ];
 
 @Table({ tableName: "recordProperties", paranoid: false })
-export class RecordProperty extends LoggedModel<RecordProperty> {
+export class RecordProperty extends Model {
+  @Column({ primaryKey: true })
+  id: string;
```

Comment:
Brining back id, createdAt, and updatedAt that we previously inherited from LoggedModel